universal-application-tool-0.0.1/test/FunctionalTest.java

```diff
@@ -24,4 +46,42 @@ public void renderTemplate() {
     assertThat("text/html").isEqualTo(html.contentType());
     assertThat(html.body()).contains("Your new application is ready.");
   }
+
+  @Test
+  public void listPersons() {
+    Http.RequestBuilder request =
+        fakeRequest(routes.PostgresController.list())
+            .header(Http.HeaderNames.HOST, "localhost:" + play.api.test.Helpers.testServerPort());
```

Comment:
Can you import this function statically like the others?

Reply:
Done.